org.mozilla.Firefox.json

```diff
@@ -0,0 +1,136 @@
+{
+  "app-id": "org.mozilla.Firefox",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.22",
+  "sdk": "org.gnome.Sdk",
+  "command": "firefox",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--filesystem=home"
+  ],
+  "modules": [
+    {
+      "name": "autoconf",
```

Comment:
Unfortunately, Firefox supports only 2.13 version of autoconf and Sdk has some newer. Using newer autoconf results in the following error:

```
========================================================================
Building module firefox in /home/vador/repos/firefox-flatpak/.flatpak-builder/build/firefox-2
========================================================================
Klone nach '/home/vador/repos/firefox-flatpak/.flatpak-builder/build/firefox-2' ...
Fertig.
Checke Dateien aus: 100% (185885/185885), Fertig.
Branch esr45 konfiguriert zum Folgen von Remote-Branch esr45 von origin.
Zu neuem Branch 'esr45' gewechselt
Applying patch firefox-makeinstall.patch
patching file GNUmakefile
Applying patch firefox-extension-dir.patch
patching file xpcom/io/nsAppFileLocationProvider.cpp
Hunk #1 succeeded at 197 (offset 10 lines).
./mach build
 0:00.25 /usr/bin/make -f client.mk -s
 0:00.26 make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
 0:00.54 client.mk:202: /run/build/firefox/obj-x86_64-unknown-linux-gnu/.mozconfig.mk: No such file or directory
 0:00.81 Clobber not needed.
 0:00.82 make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
 0:01.62 Adding client.mk options from /run/build/firefox/.mozconfig:
 0:01.62     MOZ_OBJDIR=/run/build/firefox/obj-x86_64-unknown-linux-gnu
 0:01.62     OBJDIR=/run/build/firefox/obj-x86_64-unknown-linux-gnu
 0:01.62     FOUND_MOZCONFIG=/run/build/firefox/.mozconfig
 0:01.63 /run/build/firefox/client.mk:299: *** Could not find autoconf 2.13.  Stop.
 0:01.63 client.mk:171: recipe for target 'build' failed
 0:01.63 make[1]: *** [build] Error 2
 0:01.67 0 compiler warnings present.
 0:02.68 ccache (direct) hit rate: 0.0%; (preprocessed) hit rate: 0.0%; miss rate: 100.0%
GNUmakefile:9: recipe for target 'build' failed
make: *** [build] Error 2
Error: module firefox: Der Kindprozess wurde mit Status 2 beendet
Makefile:9: die Regel für Ziel „test“ scheiterte
make: *** [test] Fehler 1
```
I can try to patch Firefox and bring the support for the latest autoconf it that sounds better for you.